### ISSUE TITLE
minify fails with ply 3.6

### ISSUE THREAD

**Issue by tomgross** [rspivak/slimit#76]
minify fails with the following error when using with ply 3.6 (released on 2015-04-26):
```
  ...
  File "/home/vagrant/Eggs/slimit-0.8.1-py2.7.egg/slimit/minifier.py", line 37, in minify
    parser = Parser()
  File "/home/vagrant/Eggs/slimit-0.8.1-py2.7.egg/slimit/parser.py", line 56, in __init__
    self.lexer.build(optimize=lex_optimize, lextab=lextab)
  File "/home/vagrant/Eggs/slimit-0.8.1-py2.7.egg/slimit/lexer.py", line 102, in build
    self.lexer = ply.lex.lex(object=self, **kwargs)
  File "/home/vagrant/Eggs/ply-3.6-py2.7.egg/ply/lex.py", line 893, in lex
    if '.' not in lextab:
TypeError: argument of type 'module' is not iterable
```

**Comment by hannseman** [comment]
Same problem here,

**Comment by rmfitzpatrick** [comment]
+1

**Comment by rmfitzpatrick** [comment]
Opened [ply 3.6 issue] (https://github.com/dabeaz/ply/issues/63) since lex.py seems to have the bug.

**Comment by byyo** [comment]
+1

**Comment by cmdelatorre** [comment]
+1 (it works with ply==3.4)

**Comment by dmoisset** [comment]
Same here, going to 3.4 did the trick. I also made it work with 3.6 forcing the lextab and yacctab arguments to be strings instead of passing the module.

The problem is easy to reproduce by doing:

```python
from slimit.parser import Parser
Parser()
```

**Comment by w00k4sh** [comment]
+1

**Comment by shemigon** [comment]
Same to me: downgrading ply to 3.4 and it works well.

**Comment by kzahel** [comment]
+1 downgraded to 3.4

**Comment by luistm** [comment]
Same to me: downgrading ply to 3.4 and it works well.

**Comment by nicolls1** [comment]
+1 downgraded to ply 3.4

**Comment by hellskitchendev** [comment]
+1 had to downgrade to ply 3.4 to get things working

**Comment by erazor83** [comment]
+1 downgraded to 3.4

**Comment by xmcp** [comment]
+1

**Comment by karyon** [comment]
fyi, this is fixed in ply 3.8, which is now on pypi, see https://pypi.python.org/pypi/ply.

**Comment by tomgross** [comment]
Works with ply 3.9
